tests/api/routes/test_events.py

```diff
@@ -105,3 +124,75 @@ def test_is_not_staff_create_event(client, reverse_url):
     assert_that(response.status_code, equal_to(HTTPStatus.UNAUTHORIZED))
     assert_that(events_before_creation, is_(empty()))
     assert_that(events_after_creation, has_length(0))
+
+
+@freeze_time("2022-10-26 23:23:23")
+@pytest.mark.django_db
+def test_list_events_endpoint_return_events(client, reverse_url):
+    event1 = (
+        EventBuilder()
+        .with_name("event1")
+        .with_description("first event description")
+        .with_meetup_link("https://www.meetup.com/event1")
+        .with_location("first event location")
+        .with_date(datetime.now() + timedelta(days=5))
+        .with_start_time(datetime.now().time())
+        .with_duration(timedelta(hours=2))
+        .build()
+    )
+    event2 = (
+        EventBuilder()
+        .with_name("event2")
+        .with_description("second event description")
+        .with_meetup_link("https://www.meetup.com/event2")
+        .with_location("second event location")
+        .with_date(datetime.now() + timedelta(days=5))
+        .with_start_time(datetime.now().time())
+        .with_duration(timedelta(hours=2))
+        .build()
+    )
+    expected_creation_datetime_str = "2022-10-26T23:23:23Z"
+    date = datetime.now() + timedelta(days=5)
+    start_time = datetime.now().time()
+    duration = timedelta(hours=2)
+    expected_payload = [
+        {
+            "id": str(event1.id),
+            "created_at": expected_creation_datetime_str,
+            "updated_at": expected_creation_datetime_str,
+            "name": event1.name,
+            "description": event1.description,
+            "meetup_link": event1.meetup_link,
+            "location": event1.location,
+            "date": "2022-10-31",
+            "start_time": "23:23:23",
+            "duration": "P0DT02H00M00S",
```

Comment:
As said in Taiga, you must return an Integer with the number of seconds.

Reply:
Ok. Done.